python/tchannel/tornado/message_factory.py

```diff
@@ -36,6 +41,156 @@ def __init__(self):
         # value: incomplete streaming messages
         self.message_buffer = {}
 
+    def build_raw_request_message(self, request, args, is_completed=False):
```

Comment:
build_raw_request_message is building message,  build_request is building request. Should not be classmethod. Self is not used in stream here, but in tracing we will.